### PR TITLE
Feature/form hook up

### PR DESCRIPTION
**Pull request by MeganHuggins**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [ ] Test

#### What's this PR do(Detailed Description)?

* This PR goes over the user form and its link to the redux store as well as the error handling for the required fields.  When the user fills out the form their information is stored in the redux-form portion of the store, to be able to more easily access this information (which is pretty nested in the redux-form layout) I have it set up to when the user clicks the 'Save' button the information is pulled down from the redux-form and placed in userForm I created in the actual redux store. This will help keep it much cleaner for the back end to use. On top of this I also set up the logic to handle any fields that are suppose to be filled out but aren't. The user is not able to move forward in the process until all the required fields are filled in and will receive errors directing them to the field they still need fill out.

#### Where should the reviewer start?

* User.js, Form.js and the Api folder in the Form folder

### CODE REVIEW

File: src/Map/accessToken.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default mapToken = 'pk.eyJ1IjoibWh1Z2dpbnMxMjY1IiwiYSI6ImNrY3RzaG1zZDIxbGwyd2xiYjA4Mm5vMmcifQ.DAx-tZndkUGJ93jAja2zgA'
```
Comment (brian-greeson) [inline src/Map/accessToken.js]: I don't think you wanted to have this in the PR?

**Review by brian-greeson — CHANGES_REQUESTED** [review]
Looks like you added your map key to the PR. Probably want to remove that.

File: src/Map/accessToken.js (hunk 1 above)
Comment (MeganHuggins) [reply src/Map/accessToken.js]: Yeah we aren't even using the map that was needed for so I'll go ahead a delete it. Thanks for catching that!

Comment (MeganHuggins) [reply src/Map/accessToken.js]: The file has been deleted and ready to merge

**Review by brian-greeson — APPROVED** [review]
Looks great to these backend eyes

**Pull request merged**